### PR TITLE
Fix Bug 1229957 - update Gigabit pages

### PR DESCRIPTION
**Pull request by mmmavis**
:tada: Done! Please let me know if I should squash my commits before or after the code preview. Thank you! :bowtie:

Bugzilla Ticket: https://bugzilla.mozilla.org/show_bug.cgi?id=1229957

### CODE REVIEW

**Comment by mmmavis** [comment]
Hi @craigcook , this PR is ready for review! It's been a while since I last contributed to this repo so please let me know if there's anything I need to change to fulfill the requirements and code standards. :smile:

**Comment by craigcook** [comment]
The `banner-bg` image can be a much smaller file as a jpeg without any loss of quality. You should also run the two new Hive logos through ImageAlpha and ImageOptim. I was able to reduce them by about 90%.

**Comment by craigcook** [comment]
The main page is a really wide column of text. Maybe the "In the news" and "Get in touch" blocks could go in a sidebar?

File: bedrock/gigabit/templates/gigabit/gigabit.html [hunk 1]
```diff
@@ -34,149 +34,110 @@ <h1 id="wordmark"><img src="{{ static('img/gigabit/gigabit-wordmark.png') }}" al
...
+        {% endtrans %}
+        </p>
+        <p>
+        {% trans %}
+          Gigabit Community Fund projects explore these new ways of reading, writing, and participating on a lightning-fast Web. Pilots supported by the Fund show current and future gigabit cities how high-speed networks can be leveraged to make learning immediate, equitable, and immersive:
+        {% endtrans %}
+        </p>
+        <ul>
+          <li>
+          {% trans url_hive_kc='//' %}
```
Comment (craigcook) [inline bedrock/gigabit/templates/gigabit/gigabit.html]: Missing URL

**Comment by mmmavis** [comment]
@craigcook thanks for the suggestion. I [updated and rebased the PR](https://github.com/mozilla/bedrock/commit/739740ed5bffa17289d9781c9cefc1af1a96c2e5) again.

**Comment by mmmavis** [comment]
Happy New Year, @craigcook :tada: :fireworks: 

I updated this PR just before the holiday break started.  Hopefully I caught and fixed all the issues - let me know if there's any other improvement I can commit, I'll get to it as quickly as I can so we can push the changes live on time before the White House event next week. Thank you! :smile:

**Comment by craigcook** [comment]
Looking good so far. Go ahead and squash your commits and I'll give it final once-over.

**Comment by mmmavis** [comment]
squashed and rebased!

**Comment by mmmavis** [comment]
Hi @craigcook ,

I wanted to rebase my PR with `master` again but got this flag. Could you tell me which line(s) I'm supposed to keep? I don't think I modified anything in `/vendor-local`... not sure how the conflict happened.  :confused: 

screen shot 2016-01-06 at 10 43 12 am

**Comment by craigcook** [comment]
This happens frequently with the legal-docs submodule. It updates along with the rebase and is marked as changed, but that change should have been excluded from the commit. It can be tricky to undo once it's already committed and I'm not sure how to explain it off the top of my head... you have to checkout the previous hash of only the submodule, then amend your commit to exclude that... or something like that. Paging @pmclanahan who can surely explain it better than I can, probably with actual steps and commands and everything.

**Comment by pmclanahan** [comment]
Here's what to do:

1. `cd vendor-local/src/legal-docs`
2. `git checkout master`
3. `git pull`
4. `cd ../../..`
5. `git add vendor-local/src/legal-docs`
6. `git commit --amend`

After doing that and a force-push to your branch the PR should no longer show an update to the submodule. This only works because currently bedrock is using the latest commit in the master branch of legal-docs, but it'd only be slightly different if that weren't the case.

**Comment by mmmavis** [comment]
Thanks for spending the time fixing the git issues with me @pmclanahan :+1: :bowtie: 

@craigcook PR is ready for your final scan!

**Comment by craigcook** [comment]
r+! :zap:

**Comment by mmmavis** [comment]
yay thank you both @craigcook and @pmclanahan :tada:

**Pull request merged**